#14325 - Remove wildcard-background from RTE

madsoulswe opened the pull request:
#14325 - Remove wildcard-background from RTE

Related issue:
https://github.com/umbraco/Umbraco-CMS/issues/14325

georgebid commented:
Hey @madsoulswe thanks for raising this ticket and then putting in a fix for it! Someone on the core collaborators team will review this soon 😄

madsoulswe commented:
Hi @georgebid !

Sorry for the late reply!

I did an update. 🙂

The transparent background is there because the tinymce theme Oxide adds a background-color to all selected elements.
So the umbraco added transparent background makes white text not be visible.

https://github.com/tinymce/oxide/blob/012346905578b0d7e59f7000d447f56243243b4a/src/less/theme/content/selection/selection.less#L50C2-L50C2


So I put back the transparent background-color and added a @blueExtraDark as text-color

I have updated the issue with more details.

georgebid commented:
@madsoulswe Ah, thanks for taking a second look 😄 I will re-test this and let you know how it goes.
